src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/DefaultTagHelperContent.cs

```diff
@@ -149,6 +149,29 @@ public override TagHelperContent AppendHtml(IHtmlContent htmlContent)
         }
 
         /// <inheritdoc />
+        public override void CopyTo(IHtmlContentBuilder destination)
+        {
+            for (var i = 0; i < Buffer.Count; i++)
+            {
+                var entry = Buffer[i];
+                if (entry == null)
```

Comment:
`HtmlContentBuilder` doesn't ever contain a null. The taghelper things do. I dunno why, but it's not part of this change.